src/Firebase.h

```diff
@@ -90,17 +92,17 @@ class FirebaseCall {
   const JsonObject& json();
 
  protected:
-  FirebaseHttpClient* http_;
+  const std::shared_ptr<FirebaseHttpClient> http_;
   FirebaseError error_;
   std::string response_;
-  DynamicJsonBuffer buffer_;
+  std::shared_ptr<StaticJsonBuffer<FIREBASE_JSONBUFFER_SIZE>> buffer_;
```

Comment:
Maybe this could simply be statically allocated as
`StaticJsonBuffer<FIREBASE_JSONBUFFER_SIZE> buffer_` without `shared_ptr`?

Reply:
I don't remember the details, but I am pretty sure latest version of JSONArduino is very strict on  memory allocations so passing this buffer around as-is results in compilation errors.